uber/decorators.py

```diff
@@ -56,7 +56,18 @@ def with_check(*args, **kwargs):
 
         if c.DEV_BOX:
             pass  # Don't redirect to any of the pages below.
-        elif c.BADGES_SOLD >= c.MAX_BADGE_SALES:
+        elif c.ATTENDEE_BADGES_SOLD >= c.MAX_BADGE_SALES:
```

Comment:
We actually have some MORE getattr magic here (... I added a lot a couple years ago) where we can ask the system if `c.ATTENDEE_BADGES_AVAILABLE` and it will automatically make these kinds of comparisons. I should probably make this _less_ magic or at least break some of that functionality out into proper function definitions. 🤔  
 
No complaints about this here, obviously, but the reason we used the old BADGES_SOLD is that the (original) intention of this was to cap at paid attendees only, with the assumption that we were more worried about being able to tell attendees we sold xty-x thousand badges vs capping things based on stocks. Maybe that's not actually important -- it certainly isn't this year!